src/functionaltests/resources/features/ttl.feature

```diff
@@ -16,3 +16,10 @@ Feature: TestTTL
     When I want to inform subscriber
     Then Verify that request has been retried
     And Check missed notification is in database
+    
+  @TestTTLforAutomaticSubscriptionTrigger
+  Scenario: Test time to live for missed notification and aggregated object with an automatic subscription trigger flow setup
+    Given A subscription is created  at the end point "/subscriptions" with non-existent notification meta 
+    And I send an Eiffel event and consequently aggregated object and thereafter missed notification is created
+    Then Based on ttl Notification document has been deleted from the database
+    And Aggregated_object document has been deleted from the database
```

Comment:
I think the "Then" statement doesnt really say "Then this happens" it's more of "Then based on" but nothing more, it sounds like a "Given" statement at the moment as it talks about a prerequisite. 

Maybe it should be more like, "Then the ttlNotification should be deleted from the database" etc etc?

Reply:
I have changed the text now..